File: travelcore/src/main/java/com/bm/travelcore/dto/BookFlightReqDTO.java
package com.bm.travelcore.dto;
import lombok.Data;
import lombok.Getter;
import java.util.List;

@Data
@Getter
public class BookFlightReqDTO {
    private String orderId;
    private String session;
    private String type;
    private List<AirOptionReqDTO> listAirOption;
    private List<PassengerReqDTO> listPassenger;
    private ContactReqDTO contact;
    private InvoiceReqDTO invoice;
    private String paymentType;
}

File: travelui/src/app/core/models/interface/book-flight-req.dto.ts
import { AirOptionReqDTO } from "./air-option-req.dto";
import { ContactInfo } from "./contact-info";
import { Invoice } from "./invoice";
import { Passenger } from "./passenger";

export interface BookFlightReqDTO {
  session: string;
  type: string;
  listAirOption: AirOptionReqDTO[];
  listPassenger: Passenger[];
  contact: ContactInfo;
  invoice: Invoice | null;
  paymentType: string;
}


